filament/src/details/Camera.h

```diff
@@ -73,13 +78,15 @@ class FCamera : public Camera {
 
     // viewing the projection matrix to be used for rendering, contains scaling/shift and possibly
     // other transforms needed by the shaders
-    math::mat4 getProjectionMatrix() const noexcept;
+    math::mat4 getProjectionMatrix(int eye = 0) const noexcept;
 
     // culling the projection matrix to be used for culling, contains scaling/shift
     math::mat4 getCullingProjectionMatrix() const noexcept;
 
+    math::mat4 getEyeFromViewMatrix(int eye) const noexcept { return mEyeFromView[eye]; }
+
     // viewing projection matrix set by the user
-    math::mat4 getUserProjectionMatrix() const noexcept { return mProjection; }
+    math::mat4 getUserProjectionMatrix() const noexcept { return mEyeProjection[0]; }
```

Comment:
That seems wrong. Now that the user can specify 2 projection matrices, this should be able to return both. This method is used in the public API, that's what `getProjectionMatrix` calls.